### ISSUE TITLE
Accessible Accordion

### ISSUE THREAD

**Issue by stevenbirr** [akserg/angular.dart.ui#95]
Hey guys,

we have very complex forms and our users would like to jump between the input elements using the [Tab] key. Unfortunately some of the UI components, like the accordion, are not accessible this way. We would like to focus the accordion header links and press [Enter] to open/close the accordion.

There is a plugin such as
http://paypal.github.io/bootstrap-accessibility-plugin/demo.html

Could you please add such options, e.g. using aria-attributes, in your library? Is it possible with low effort?

Regards, 
Steven

**Comment by akserg** [comment]
Hi Steven.
Is this issue still valuable for you project?

**Comment by stevenbirr** [comment]
Yes, it is, would be great! Default Bootstrap works like a charm. http://getbootstrap.com/javascript/#collapse
